<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecorderGUI</class>
 <widget class="QFrame" name="RecorderGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChannelGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	border-width: 5px;
	border-style: inset;
	border-radius: 20px;
	background: url(:/images/channels.png);
}

QLabel
{
	color: white;
	border: none;
	background:none;
}

QLabel#labelTimecode,
QLabel#labelDuration,
QLabel#labelRecordPath,
QLineEdit#lineEditClipName,
QPushButton#pushButtonTimecode
{
	border-width: 2px;
	border-radius: 10px;
	border-style: solid;
	border-color: grey;
	color: black;
	background:white;
}

QPushButton#pushButtonStart,
QPushButton#pushButtonStop,
QPushButton#pushButtonClear
{
	background-color: lightGrey;
	border-style: outset;
	border-width: 10px;
	border-color: grey;
	border-radius: 20px;
}
QPushButton#pushButtonStart:pressed,
QPushButton#pushButtonStop:pressed,
QPushButton#pushButtonClear:pressed
{
	border-style: inset;
}

QFrame#videoFrame
{
	border-width: 2px;
	border-radius: 0px;
	background: none;
	border-style: solid;
	border-color: grey;
}

QLabel#labelSignalError, QLabel#labelNoRef
{
	color: red;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRecorder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>R
e
c
o
r
d
e
r</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QVUMeter" name="audioMeter" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>81</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClipName">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>14</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Insert Clip Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="videoFrame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>388</width>
     <height>220</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="PreviewerRGB" name="preview" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>384</width>
      <height>216</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>42</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTimecode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>42</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00:00</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSignalError">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>300</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Signal Error</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRecordPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>C:\</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPath">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>299</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChoosePath">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>299</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVUMeter</class>
   <extends>QWidget</extends>
   <header>qvumeter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewerRGB</class>
   <extends>QWidget</extends>
   <header>previewerrgb.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
